wp/plugin/wp.ml

```diff
@@ -332,7 +333,8 @@ module Cmdline = struct
       ~doc:"Amount of times to unroll each loop. By default, wp will unroll each \
             loop 5 times."
 
-  let output_vars = param (list string) "output-vars" ~default:["RAX"; "EAX"]
+  let compare_final_reg_values = param (list string) "compare-final-reg-values"
```

Comment:
Why "final" rather than "output"?

Reply:
I think the word final is used because the registers that are being compared aren't necessarily going to be the output registers, but will be the final value of any register.

Reply:
How about `compare_post_reg_values`?

Reply:
I have no problems with `compare_post_reg_values`.